darts/utils/timeseries_generation.py

```diff
@@ -203,13 +208,14 @@ def random_walk_timeseries(length: int = 10,
     times = pd.date_range(periods=length, freq=freq, start=start_ts)
     values = np.cumsum(np.random.normal(mean, std, size=length))
 
-    return TimeSeries.from_times_and_values(times, values, freq=freq)
+    return TimeSeries.from_times_and_values(times, values, freq=freq, columns=pd.Index([column_name]))
 
 
 def holidays_timeseries(time_index: pd.DatetimeIndex,
                         country_code: str,
                         prov: str = None,
-                        state: str = None) -> TimeSeries:
+                        state: str = None,
+                        column_name: Optional[str] = '0') -> TimeSeries:
```

Comment:
[nitpick]: actually "holidays" might be a better name here?

Reply:
Actually more generally we could create s specific name for each of these generations ('sine', 'constant' etc). What do you think?

Reply:
Good idea!

Reply:
Yep makes sense! A lot of tests and notebooks will have to be changed though, just as a heads-up

Reply:
Yes, and in the tests, we have things such as 

series = linear_timeseries(start_value=1, length=12, freq='MS', start_ts=pd.Timestamp('2000-01-01'), end_value=12) 
zeroes = constant_timeseries(value=0.0, length=12, freq='MS', start_ts=pd.Timestamp('2000-01-01'))

def function(ts, x):
     return x - ts.month

new_series = series.map(function)
self.assertEqual(new_series, zeroes)` 

the assertEqual here doesnt work anymore, as the column names are different ('const' and 'linear'). It seems important to be able to quickly change names of the columns, by adding a .withColumnName property to the TImeSeries object to create a new timeserie with column name changed (immutability) so I can change the name of the column before testing the assertEqual.
wdyt?

Reply:
It would look like :

my_ts = my_ts.withColumnRenamed('old_col', 'new_col')

exactly like pyspark